guides/common/modules/proc_renaming-smart-proxy.adoc

```diff
@@ -55,64 +55,9 @@ For example, to copy the archive file to the `root` user's home directory:
 Ensure that you enter the full path to the `.tar` file.
 . If you have created a custom certificate for your {SmartProxyServer}, deploy the certificate to your {SmartProxyServer} by entering the `{foreman-installer}` command that the `{certs-generate}` command returned in a previous step.
 For more information, see {InstallingSmartProxyDocURL}deploying-a-custom-ssl-certificate-to-{smart-proxy-context}-server_{smart-proxy-context}[Deploying a Custom SSL Certificate to {SmartProxyServer}] in _{InstallingSmartProxyDocTitle}_.
-. On all hosts registered to your {SmartProxyServer}, enter the following commands to reinstall the bootstrap RPM, reregister clients, and refresh their subscriptions.
+. Reregister all hosts that are registered to your {SmartProxyServer}.
```

Comment:
```suggestion
. Re-register all hosts that are registered to your {SmartProxyServer}.
```

Reply:
Applied.

Reply:
Ah, I see that your peer reviewer requested the change. 

@Lennonka I will share the link with you privately.